api/all/src/main/java/io/opentelemetry/api/OpenTelemetry.java

```diff
@@ -113,6 +116,52 @@ default MeterBuilder meterBuilder(String instrumentationScopeName) {
     return getMeterProvider().meterBuilder(instrumentationScopeName);
   }
 
+  /**
+   * Returns the {@link LoggerProvider} for this {@link OpenTelemetry}.
+   *
+   * <p>The OpenTelemetry logging API exists to enable the creation of log appenders, which bridge
+   * logs from other log frameworks (e.g. SLF4J, Log4j, JUL, Logback, etc) into OpenTelemetry via
+   * {@link Logger#logRecordBuilder()}. It is <b>NOT</b> a replacement log framework.
+   *
+   * @since 1.26.0
+   */
+  default LoggerProvider getLoggerProvider() {
```

Comment:
I don't know about adding this here. I know it's supposed to be first-class, but if we add something called this to OpenTelemetry, _everyone_ is going to think this is for logging, not for building appenders. 

I think this is a bad idea.

Reply:
It is a bit weird isn't it?

We'd have to keep a separate GlobalLoggerProvider around.

We do already include SdkLoggerProvider in OpenTelemetrySdk. Would you aim to unwind that for consistency?

Reply:
I've renamed this to `LoggerProvider getLogsBridge()`, which along with poor user ergonomics and thorough documentation should be enough to discourage use by end users.